### ISSUE TITLE
json ld validation errors

### ISSUE THREAD

**Issue by seewodg** [uncefact/spec-jsonld#197]
@onthebreeze I noted some validation errors in the untp-core JSON LD:

When I validate the JSON LD at: [schema.org json validator](https://validator.schema.org/)

I get some errors for: https://jargon.sh/user/unece/untp-core/v/working/artefacts/jsonld/render.jsonld.   

It appears the issue is a lack of @type in relation to schema:rangeIncludes and schema:domainIncludes.

**Comment by onthebreeze** [comment]
@colugo - one more for you.

**Comment by seewodg** [comment]
@onthebreeze While we are onto identifiers, untp-core:Identifier seems like it should be a parent class for untp:FacilityID. Currently untp-core:FacilityID is defined in the json ld as:

    {
      "@id": "untp-core:FacilityID",
      "@type": "rdfs:Class",
      "rdfs:label": "FacilityID",
      "schema:source": {
        "@id": "https://test.uncefact.org/vocabulary/untp/core/FacilityID"
      },
      "rdfs:comment": "The ID and Name of a facility. Used for lightweight links from other classes."
    }

Whereas it more logically would be described as:

    {
      "@id": "untp-core:FacilityID",
      "@type": "rdfs:Class",
      "rdfs:label": "FacilityID",
      "schema:source": {
        "@id": "https://test.uncefact.org/vocabulary/untp/core/FacilityID"
      },
      "rdfs:subClassOf": "untp-core:Identifier",
      "rdfs:comment": "The ID and Name of a facility. Used for lightweight links from other classes."
    }

**Comment by onthebreeze** [comment]
Yes. Changed it so that "product", "facility" etc have parent "entity"
